src/olympia/addons/models.py

```diff
@@ -743,23 +745,23 @@ def update_version(self, ignore=None, _signal=True):
         Pass ``_signal=False`` if you want to no signals fired at all.
 
         """
-        current = self.find_latest_public_version()
-        latest = self.find_latest_version(ignore=ignore)
+        new_current_version = self.find_latest_public_listed_version()
+        new_latest_version = self.find_latest_version(ignore=ignore)
```

Comment:
how close are we to dropping the latest_version property on Addon?  That it's not per channel is concerning.